Rido.IoTHubClient.Tests/HubMqttClientFixture.cs

```diff
@@ -213,7 +213,19 @@ public async Task ConnectModuleWithCertAndModelId()
             await client.CloseAsync();
         }
 
-
+        [Fact]
+        public async Task ConnectSameDeviceTwiceTriggersDisconnect()
```

Comment:
@wickste 
this test connects two clients with the same id, hence the first one disconnects.